eleventy.config.ts

```diff
@@ -54,9 +54,10 @@ export default async function(eleventyConfig: UserConfig) {
     eleventyConfig.addGlobalData('runMode', runMode);
   });
 
-  let watch;
   eleventyConfig.on('eleventy.before', function({ runMode }) {
-    watch ||= runMode === 'watch' && $({ stdout: ['pipe'], stderr: ['pipe'] })`npx tspc -b --watch`;
+    (() => {
+      return runMode === 'watch' && $({ stdout: ['pipe'], stderr: ['pipe'] })`npx tspc -b --watch`;
```

Comment:
This will spawn a new tsc watch process on each build

Reply:
good thinking, was thinking the opposite at the time, reverted